src/figuresClasses.ts

```diff
@@ -1,19 +1,93 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
+  getArea: () => number;
 }
 
-export class Triangle {
+export function rounded(area: number): number {
+  let areaFixed = area;
+
+  if (areaFixed !== +areaFixed.toFixed(3)) {
+    areaFixed = +areaFixed.toFixed(3).slice(0, -1);
```

Comment:
try to use `Math.floor` here to make code more clear

Reply:
It is not work enough for test. In test I must do 556.55, but if I use Math.floor, I will have 556.

Reply:
And if I try 'round floor' the last number, I must use more code then this.

Reply:
it can be done using something like this `Math.floor(number * 100) / 100` 🙂

Reply:
Aaaa, Thanks)